bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/eclipse/common/core/model/RatingGroup.java

```diff
@@ -37,6 +39,22 @@ public String getDisplayName() {
 		return this.displayName;
 	}
 
+	@Override
+	/**
+	 * @since 2.7
+	 */
+	public String getLanguageSensitiveDisplayName(String language) {
+		// TODO Auto-generated method stub
+		switch (language) {
+		case "English":
+			return engDisplayName;
+		case "Deutsch":
```

Comment:
Language strings should either be constants or enum.

Reply:
Or can you derive these values from the locale?

Reply:
as ratingGroup changes are reversed, comment is now irrelevant, but was done like this in MistakeType.java